src/model/guild/mod.rs

```diff
@@ -626,7 +626,7 @@ impl Guild {
     pub async fn create(
         http: impl AsRef<Http>,
         name: &str,
-        icon: Option<&str>,
+        icon: ImageHash,
```

Comment:
Why was this changed to a plain `ImageHash`? Guilds can be created without an icon, so this should stay as `Option`al.

```suggestion
        icon: Option<ImageHash>,
```

Reply:
Ah, guess this was a mistake